Hortali-App/Shared/Design System/Views/Components/CollectionGroup/CollectionGroup.swift

```diff
@@ -36,6 +36,13 @@ public class CollectionGroup: UIView {
         return col
     }()
     
+    internal var emptyView = {
+        let view = EmptyView()
+        view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Se vc já está criando um componente do zero, essa linha não faz sentido estar na hora de instanciar o componente, principalmente que toda vez que for usar esse componente vai ter que fazer essa configuração.

Coloque essa linha no próprio inicializador do componente, ou seja: coloque dentro do `init()` da `EmptyView()`